warehouse/query-core/src/test/java/datawave/query/HitsAreAlwaysIncludedCommonalityTokenTest.java

```diff
@@ -178,14 +176,13 @@ protected void runTestQuery(Connector connector, String queryString, Date startD
             if (hitAttribute instanceof Attributes) {
                 Attributes attributes = (Attributes) hitAttribute;
                 for (Attribute attr : attributes.getAttributes()) {
-                    if (attr instanceof Content) {
-                        Content content = (Content) attr;
-                        Assert.assertTrue(goodResults.contains(content.getContent()));
+                    if (TypeAttribute.matches(attr, HitTermType.class)) {
+                        String content = attr.toString();
```

Comment:
Ivan's comment about relying on `getData()` instead of `toString()` should apply here too, I think